assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx

```diff
@@ -23,7 +20,7 @@ const TableView: React.FC = () => {
 			filterState={filterState}
 			bodyRowGenerator={bodyRowGenerator}
 			headerRowGenerator={headerRowGenerator}
-			className={tableClassName}
+			className={'ee-dates-list-list-view ee-fade-in'}
```

Comment:
`list-list` :)
`list-list-list`

Reply:
oh I didn't even notice that LOLZ
that likely should have been changed to `ee-dates-list-table-view`

will fix in my next cleanup PR